lib/EnhancedSwitch.js

```diff
@@ -247,9 +247,11 @@ var EnhancedSwitch = (function (_React$Component) {
       }
     }
   }, {
-    key: 'isChecked',
-    value: function isChecked() {
-      return this.refs.checkbox.checked;
+    key: 'setIndeterminate',
+    value: function setIndeterminate() {
+      if (this.props.indeterminate) {
+        this.refs.checkbox.indetermiante = true;
```

Comment:
"indetermiante"? This is broken -- there is no "indetermiante" property on dom checkbox elements. Now, "indeterminate" is a different story...